pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -197,30 +200,69 @@ func (c *Cluster) createSelfSignedCert() (bool, error) {
 	return false, nil
 }
 
+func LatestCephVersions(c *client.ClusterInfo) bool {
```

Comment:
Write unit tests for this and give it a proper name for more context.

Reply:
Added unit test and tried to give a better name with the comment above for more context.